go/vt/vttablet/tabletmanager/vreplication/vreplicator.go

```diff
@@ -43,11 +43,15 @@ var (
 	// between the two timeouts.
 	idleTimeout = 1100 * time.Millisecond
 
-	dbLockRetryDelay    = 1 * time.Second
-	relayLogMaxSize     = flag.Int("relay_log_max_size", 250000, "Maximum buffer size (in bytes) for VReplication target buffering. If single rows are larger than this, a single row is buffered at a time.")
-	relayLogMaxItems    = flag.Int("relay_log_max_items", 5000, "Maximum number of rows for VReplication target buffering.")
-	copyTimeout         = 1 * time.Hour
-	replicaLagTolerance = 10 * time.Second
+	dbLockRetryDelay = 1 * time.Second
+	relayLogMaxSize  = flag.Int("relay_log_max_size", 250000, "Maximum buffer size (in bytes) for VReplication target buffering. If single rows are larger than this, a single row is buffered at a time.")
+	relayLogMaxItems = flag.Int("relay_log_max_items", 5000, "Maximum number of rows for VReplication target buffering.")
+
+	copyTimeout              = 1 * time.Hour
+	copyPhaseDurationSeconds = flag.Int("vreplication_copy_phase_duration", 3600, "Duration (seconds) for each copy phase loop (before running the next catchup)")
+
+	replicaLagTolerance        = 60 * time.Second
+	replicaLagToleranceSeconds = flag.Int("vreplication_replica_lag_tolerance", 60, "Replica lag threshold (seconds): once lag is below this we switch from copy phase to the replication (streaming) phase")
```

Comment:
again, naming-wise, either call this `vreplication_replica_lag_tolerance_seconds` or make it a `flag.Duration()`.